src/js/media-chrome-listbox.js

```diff
@@ -7,6 +7,28 @@ template.innerHTML = `
 <style>
   :host ul {
     list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+    margin: 0;
+    padding: 0.5em;
+    /* background-color: var(--media-control-background, rgba(20,20,30, 0.7)); */
+    background-color: black;
```

Comment:
don't forget to revert this to use the CSS var

Reply:
When I was first trying those other colors, there were issues around stacking transparencies and generally not looking great. However, definitely worth revisiting this.

Reply:
Went back to this one, but tweaked the color of the selected item a bit so that contrast ratio between it and unselected items is higher.